lib/components/artist/articles/article.js

```diff
@@ -12,13 +12,16 @@ import ImageView from '../../opaque_image_view';
 class Article extends React.Component {
   render() {
     const article = this.props.article
+    // TODO The outer wrapping view is currently only there because setting `marginRight: 20` on the Article from the
```

Comment:
Yeah this is a weird thing - I find that setting the `style` of an entire component from the parent gives mixed results. I should read up on how flexbox reconciles self-styling and styling-by-parent.